admin/scss/bootstrap/_variables.scss

```diff
@@ -38,6 +38,16 @@ $brand-success:             #3fa142;
 // $brand-warning:             #f0ad4e;
 $brand-danger:              #D40404;
 
+// Backgrounds
+$background-main: 				#f6f7f8;
```

Comment:
Either we correctly align all values here in terms of whitespace, or we admit defeat and only use a single whitespace after the double colon. We shouldn't mix both approaches, it looks even messier ;)